lib/safira_web/controllers/redeemable_controller.ex

```diff
@@ -2,16 +2,32 @@ defmodule SafiraWeb.RedeemableController do
   use SafiraWeb, :controller
 
   alias Safira.Store
+  alias Safira.Accounts
 
   action_fallback(SafiraWeb.FallbackController)
 
   def index(conn, _params) do
-    redeemables = Store.list_redeemables()
+    redeemables = Store.list_store_redeemables()
     render(conn, "index.json", redeemables: redeemables)
   end
 
   def show(conn, %{"id" => id}) do
     redeemable = Store.get_redeemable!(id)
     render(conn, "show.json", redeemable: redeemable)
   end
+
+  def my_items(conn, _params) do
+    attendee = Accounts.get_user(conn)
+                |> Map.fetch!(:attendee)
+    cond do
+      not is_nil(attendee) -> 
+        redeemables = Store.get_attendee_redeemables(attendee)
+        render(conn, "index_my_redeemables.json", redeemables: redeemables)
+      true ->
+        conn
+          |> put_status(:unauthorized)
+          |> json(%{error: "Only attendees can have products!"})
+    end
+  end
```

Comment:
I think this should be moved to attendeeView like badges and prizes, because you're preloading this info in `Safira.Accounts.get_attendee!`. Then you should remove this function. This way, when the attendee sees its profile, all your information is there

Reply:
I just saw that you already have this on AttendeeView, so I don't think this function will be necessary.

Reply:
I wanted a way to quickly see "what I have" without going to the profile, but that's fine. Deleting is always quicker than creating, right? 😂

Reply:
And it's easy to add the quantity